client/src/pages/MealPage/sections/MealCookInfo/MealCookInfo.js

```diff
@@ -49,7 +56,10 @@ MealCookInfo.propTypes = {
       cook_lastname: PropTypes.string,
       cook_summary: PropTypes.string,
       cook_description: PropTypes.string,
-      is_thumbnail: PropTypes.arr
+      cook_image_url: PropTypes.string,
+      is_thumbnail: PropTypes.arr,
+      av_reviews: PropTypes.string,
```

Comment:
same

Reply:
This cannot be changed without changing the name in the db which we agreed we wouldn't do.